File: rb/utility/make_context.rb
# MentalitySkillTraining SDK utility: make_context
require_relative '../core/context'
module MentalitySkillTrainingUtilities
  MakeContext = ->(ctxmap, basectx) {
    MentalitySkillTrainingContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// MentalitySkillTraining SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class MentalitySkillTrainingMakeContext
{
    public static function call(array $ctxmap, ?MentalitySkillTrainingContext $basectx): MentalitySkillTrainingContext
    {
        return new MentalitySkillTrainingContext($ctxmap, $basectx);
    }
}
